DownUnder.UI/UI/Widgets/DataTypes/UIInputState.cs

```diff
@@ -1,75 +1,74 @@
 ﻿using System.Diagnostics;
-using DownUnder.UI.Input;
-using DownUnder.UI.Utilities.CommonNamespace;
 using Microsoft.Xna.Framework;
 using Microsoft.Xna.Framework.Input;
 using MonoGame.Extended;
+using DownUnder.UI.Input;
+using DownUnder.UI.Utilities.CommonNamespace;
 
 //Todo: Trigger inputs shopuld not have set parameters
-namespace DownUnder.UI.UI.Widgets.DataTypes
-{
+namespace DownUnder.UI.UI.Widgets.DataTypes {
     /// <summary> Used to communicate to widgets input. Can be updated or set manually. </summary>
-    public class UIInputState {
-        private Vector2 _previous_mouse_wheel_scroll = new Vector2(Mouse.GetState().HorizontalScrollWheelValue, Mouse.GetState().ScrollWheelValue);
-        private MouseState _mouse_state = Mouse.GetState();
-        private MouseState _previous_mouse_state = Mouse.GetState();
-        private KeyboardState _keyboard_state = Keyboard.GetState();
-        private KeyboardState _previous_keyboard_state = Keyboard.GetState();
-        private BufferedKeyboard buffered_keyboard = new BufferedKeyboard();
-        private bool _insert_key_down = false;
-        private bool _previous_insert_key_down = false;
+    public sealed class UIInputState {
+        Vector2 _previous_mouse_wheel_scroll = new Vector2(Mouse.GetState().HorizontalScrollWheelValue, Mouse.GetState().ScrollWheelValue);
+        MouseState _mouse_state = Mouse.GetState();
+        MouseState _previous_mouse_state = Mouse.GetState();
+        KeyboardState _keyboard_state = Keyboard.GetState();
+        KeyboardState _previous_keyboard_state = Keyboard.GetState();
+        readonly BufferedKeyboard buffered_keyboard = new BufferedKeyboard();
+        bool _insert_key_down;
+        bool _previous_insert_key_down;
```

Comment:
Private is pretty useless outside of class/struct use despite being one of the the first things we're taught